FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -60,17 +65,29 @@ else
                             <FluentIcon Color="Color.Disabled" Value=@(new Icons.Regular.Size16.PersonBoard())/>@_recipe.Servings
                         </FluentBadge>
                     </FluentOverflowItem>
-                    @foreach (var item in _tagNames)
+                    @foreach (var item in _recipe.Tags)
                     {
                         <FluentOverflowItem>
-                            <FluentBadge Appearance="Appearance.Accent">@item</FluentBadge>
+                            <FluentBadge Appearance="Appearance.Accent">@item.Name</FluentBadge>
                         </FluentOverflowItem>
                     }
                 </FluentOverflow>
+                @if (_author != null)
+                {
+                    <FluentAnchor Href=@($"/users/{_author.Id}") Appearance="Appearance.Stealth" Style="height: 45px">
+                        <FluentStack Orientation="Orientation.Horizontal" VerticalAlignment="VerticalAlignment.Center">
+                                <FluentLabel Typo="Typography.Body">By </FluentLabel>
```

Comment:
Perhaps "Author", "Authored by" or "Created by" would better illustrate the data, but this works too

Reply:
I wanted to keep it short in horizontal width